src/contracts/webauthn_account_contract/src/main.nr

```diff
@@ -125,10 +125,11 @@ contract WebAuthnAccount {
             // TODO use challenge from params
```

Comment:
I wonder what this one is about? Would you elaborate?

Reply:
There is a `challenge: Field` param, and in final implementation this should be used and converted to `[u8; 32]`

There is a task for this: https://github.com/orgs/zkworks-xyz/projects/2?pane=issue&itemId=43423555